crates/shared/src/price_estimation/native_price_cache.rs

```diff
@@ -14,7 +14,11 @@ use tracing::Instrument;
 struct Metrics {
     /// native price cache hits misses
     #[metric(labels("result"))]
-    native_price_cache: IntCounterVec,
+    native_price_cache_access: IntCounterVec,
+    /// number of items in cache
+    native_price_cache_size: IntGauge,
```

Comment:
Should we maybe also add a gauge that tracks the number of outdated prices during each update cycle?
Then we could see when we hit a point of diminishing returns when we increase the throughput with configuration changes.

Reply:
added